pallets/asset-registry/src/tests.rs

```diff
@@ -206,3 +208,54 @@ fn set_metadata_works() {
 			);
 		});
 }
+
+#[test]
+fn update_asset() {
```

Comment:
These tests actually dont really test the the storage was updated. They only check the result of the update call.

Reply:
added